src/components/ec-timer/ec-timer.vue

```diff
@@ -78,18 +93,38 @@ const diameter = computed(() => radius.value * 2 + strokeWidth.value);
 const viewbox = computed(() => `0 0 ${diameter.value} ${diameter.value}`);
 const circumference = computed(() => 2 * Math.PI * radius.value);
 const steps = computed(() => circumference.value / props.seconds);
-const offset = computed(() => circumference.value + steps.value * secondsLeft.value);
+const offset = computed(() => circumference.value + steps.value * totalSecondsLeft.value);
 
 const radius = ref(24);
 const strokeWidth = ref(4);
-const secondsLeft = ref(props.seconds);
+const minutesLeft = computed(() => {
+  if (props.showMinutes) {
+    return Math.floor(totalSecondsLeft.value / 60);
+  }
+
+  return null;
+});
+const secondsLeft = computed(() => {
+  if (!props.isRunning) {
+    return props.seconds;
+  }
+  if (props.showMinutes) {
+    return totalSecondsLeft.value - Math.round(minutesLeft.value * 60);
+  }
+  return totalSecondsLeft.value;
+});
+const totalSecondsLeft = ref(getTotalSecondsLeft(props.seconds));
```

Comment:
```suggestion
const totalSecondsLeft = ref(props.seconds);
```

Reply:
it breaks the logic when isRunning is false